coalib/bears/requirements/PythonRequirement.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bears.requirements.PackageRequirement import PackageRequirement
+
+
+class PythonRequirement(PackageRequirement):
+    """
+    This class is a subclass of ``PackageRequirement``, and helps specifying
+    requirements from ``pip``, without using the manager name.
+    """
+
+    def __init__(self, package, version=""):
+        """
+        Constructs a new PythonRequirement, using the PackageRequirement
+        constructor.
```

Comment:
maybe also a little doctest like in the commit before, that tests the members of this class :+1:, this also demonstrates the attribute access (meaning how members are named)